packages/features/ee/teams/lib/payments.ts

```diff
@@ -121,15 +127,19 @@ export const purchaseTeamOrOrgSubscription = async (input: {
   let priceId: string | undefined;
 
   if (pricePerSeat) {
-    const customPriceObj = await getPriceObject(fixedPrice);
-    priceId = await createPrice({
-      isOrg: !!isOrg,
-      teamId,
-      pricePerSeat,
-      billingPeriod,
-      product: customPriceObj.product as string, // We don't expand the object from stripe so just use the product as ID
-      currency: customPriceObj.currency,
-    });
+    if (isOrg && pricePerSeat === ORGANIZATION_SELF_SERVE_PRICE && seats < ORGANIZATION_MIN_SEATS) {
```

Comment:
Why do we need the seats to be less than minimum seats? I think that's only possible in case ADMIN provides the seats, otherwise seats can't go less than the minimum right?